src/neo-vm/Debugger.cs

```diff
@@ -71,15 +71,15 @@ public VMState StepOut()
 
         public VMState StepOver()
         {
-            if (engine.State.HasFlag(VMState.HALT) || engine.State.HasFlag(VMState.FAULT))
+            if (engine.State == VMState.HALT || engine.State == VMState.FAULT)
                 return engine.State;
             engine.State &= ~VMState.BREAK;
```

Comment:
I think that should be faster and cleaner change this sentences
```suggestion
            engine.State = VMState.NONE;
```